### PR TITLE
[Terraform] Replace hardcoded unique names with optional configurable variables

### PR DESCRIPTION
**Pull request by 119Vik**
## Summary

Updating current terraforms modules to make them more configurable. These changes will allow users to do several deployments of Orc8r on top of the same EKS cluster

### CODE REVIEW

File: orc8r/cloud/deploy/terraform/orc8r-helm-aws/logging.tf [hunk 1]
```diff
@@ -18,7 +18,7 @@ locals {
 resource "helm_release" "fluentd" {
   count = var.elasticsearch_endpoint == null ? 0 : 1
 
-  name       = "fluentd"
+  name       = var.external_dns_deployment_name
```
Comment (mpgermano) [inline orc8r/cloud/deploy/terraform/orc8r-helm-aws/logging.tf]: This should be `var.fluentd_deployment_name` right?

**Review by mpgermano — CHANGES_REQUESTED** [review]
Looks good. Please correct the variable name for fluentd then I will approve

File: orc8r/cloud/deploy/terraform/orc8r-helm-aws/logging.tf (hunk 1 above)
Comment (119Vik) [reply orc8r/cloud/deploy/terraform/orc8r-helm-aws/logging.tf]: yes. 100%

**Review by hcgatewood — COMMENTED** [review]
Clearing review queue, deferring to @mpgermano's review

**Review by mattymo — APPROVED** [review]

**Review by mpgermano — APPROVED** [review]

**Pull request merged**